src/test/java/au/org/democracydevelopers/raireservice/testUtils.java

```diff
@@ -25,10 +25,11 @@ the raire assertion generation engine (https://github.com/DemocracyDevelopers/ra
 import au.org.democracydevelopers.raireservice.persistence.entity.Assertion;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.reflect.TypeToken;
-import java.math.BigDecimal;
+import java.util.ArrayList;
 import java.util.List;
 
```

Comment:
Add class level comment.

Some lines in this file are too long -- use vertical line guide.